tools/Runtime Condenser/condenser.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# Runtime Condenser, rewritten in python by tkdrg
+# Only the first set of proc, file, usr and src will be kept. The rest is truncated.
+# This script is valid on both python2 and python3
+runtimes = dict()
+infloops = dict()
+harddels = dict()
+
+
+def readFromFile(target='input.txt'):
+	global runtimes, infloops, harddels
+	f = open(target)
+
+	for line in iter(f.readline, ''):
```

Comment:
ah good point, I didn't consider that.

I'm not sure if it reflects updates in the cursor